integration/config/core.yaml

```diff
@@ -3,7 +3,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 #
-# Based on sampleconfig from github.com/hyperledger/fabric.git tagged with v2.0.1 
+# Based on sampleconfig from github.com/hyperledger/fabric.git tagged with v2.1.0 
```

Comment:
Do other config at same dir with same comment need to be updated?

Reply:
good catch: i thought i grepped for all files containing 2.0.1 but evidently screwed up :-)

Reply:
there is now a corresponding fixup

Reply:
not screw up, things happen, thanks for extra fixup.